app/ammunition/armours.py

```diff
@@ -0,0 +1,4 @@
+class Armour:
+    def __init__(self, part: str, protection: int) -> None:
+        self.part = part
+        self.protection = protection
```

Comment:
no need to make separate module for every calss of equip, u can implement them together

Reply:
I make one module equip and moved all equipment to it